### PR TITLE
Added Composer JSON file.

### PR DESCRIPTION
**Pull request by pnikolov**
Fixes https://github.com/fgnass/spin.js/issues/319

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "fgnass/spin.js",
+  "description": "An animated CSS3 loading spinner with VML fallback for IE.",
+  "license": "Licensed under MIT",
```
Comment (TimothyGu) [inline composer.json]: Just MIT is fine.

**Comment by pnikolov** [comment]
Latest commit sorts out the _license_ name, also adds a _homepage_ and more _keywords_.

**Comment by TimothyGu** [comment]
Thanks!

**Comment by craue** [comment]
The package should be registered on https://packagist.org as well.

**Comment by pnikolov** [comment]
Yes, I've minted this in #319 
Submit URL:
https://packagist.org/packages/submit

**Comment by craue** [comment]
Just trying again... :smirk:
/ping @fgnass

**Pull request merged**